sklearn/base.py

```diff
@@ -292,6 +293,24 @@ def __repr__(self):
         return '%s(%s)' % (class_name, _pprint(self.get_params(deep=False),
                                                offset=len(class_name),),)
 
+    def __getstate__(self):
+        if type(self).__module__.startswith('sklearn.'):
+            return dict(self.__dict__.items(), __version__=__version__)
+        else:
+            return dict(self.__dict__.items())
```

Comment:
I'm not fundamentally against adding it as an attribute to ``BaseEstimator`` but I'd rather make the attribute private to not pollute the autocomplete too much.